php/libraries/NDB_Config.class.inc

```diff
@@ -238,43 +238,6 @@ class NDB_Config
         return $retVal;
     }
 
-    /**
-     * Attempts to determine the site of the user currently logged in and uses
-     * that to get site specific settings and override study defaults,
-     * building the _siteSettings property
-     *
-     * @return void (but modifies $this->siteSettings)
-     */
-    function mergeSettings()
-    {
-        // make sure this is a web client
-        if (!class_exists("User") || !isset($_SESSION['State'])) {
-            return;
-        }
-
-        // make sure a user has logged in
-        $username = $_SESSION['State']->getUsername();
-        if (empty($username)) {
-            $this->_siteSettings =& $this->_settings;
-        } else {
-            $user     =& User::singleton($username);
-            $siteName = Utility::getCleanString($user->getSiteNames());
-            if (isset($this->_settings['sites'][$siteName])
```

Comment:
In my tests this condition was never true, meaning the function simply does line 273, 

```php
$this->_siteSettings =& $this->_settings;
```

Meaning that _siteSettings is always the same as _settings and therefore is unnecessary.